hardhat/test/unit/21_Utils.t.js

```diff
@@ -0,0 +1,122 @@
+const {
+  loadFixture,
+} = require("@nomicfoundation/hardhat-toolbox/network-helpers");
+const { expect } = require("chai");
+const hre = require("hardhat");
+const { ethers } = hre;
+const { fixtures } = require("../helper_fixture");
+
+// describe.only("Utils", function () {
```

Comment:
remove commented code

Reply:
Really need to remove ? It's to ease change when working on tests